common/common.go

```diff
@@ -282,6 +286,15 @@ func RoundFloat(x float64, prec int) float64 {
 	return rounder / pow
 }
 
+// CountInt counts how many digits in an integer and returns the count
```

Comment:
This won't count 0 as being a digit

Reply:
Why would you pass in zero and require a digit count? It's just going to be zero.